p/themes/base-theme/template.css

```diff
@@ -74,12 +74,12 @@ sup {
 
 kbd {
 	background-color: #eee;
-	padding: 2px 4px;
+	padding: 2px 4px 2px 24px;
 	display: inline-block;
 	color: #333;
 	border: 1px solid #b4b4b4;
 	border-radius: 3px;
-	white-space: nowrap;
```

Comment:
For clarity it might be preferable to replace this with `pre-wrap` instead of what's proposed in this PR, although I do like the look with indentation.

Reply:
Incidentally, that'd also take care of everything automagically. You'll still have to add for example `overflow-wrap: anywhere` and/or `overflow: auto` to make this PR fully do what it promises. (I.e., try with `abunchofreallylongtextwithoutaslashorsomething`).